.jazzy.yml

```diff
@@ -0,0 +1,50 @@
+# jazzy --swift-version 2.2 -m Deferred -g "https://github.com/bignerdranch/Deferred" -a "Big Nerd Ranch" -u "http://bignerdranch.github.io" --module-version=2.0.0 -r "http://bignerdranch.github.io/Deferred/"
```

Comment:
For scripts, I prefer to use the long version of options where possible. The abbreviation when typing them out yourself of short options isn't needed, and the explanatory power of long options is wonderful.

If jazzy'll let you, I often spread args in a group across lines using continuation backslashes. (This happens a lot with gnarly `curl` commands.) Here's an example of what I mean from a Makefile I have lying around:

```make
run:
	browser-sync start \
		--server \
		--files "stylesheets/*.css, scripts/*.js, *.html"
```